README.md

```diff
@@ -21,7 +21,7 @@ with pipes() as (out, err):
 stdout = out.read()
 ```
 
-Capture stdout/stderr in StringIO:
+Capture stdout/stderr in StringIO and forward C-level stderr to Python sys.stdout:
```

Comment:
This is not quite accurate. `stderr=STDOUT` means that stderr is captured to the same stream as stdout, which is not sys.stdout in this case.

Maybe this:

```suggestion
Capture both stdout and stderr in a single StringIO object:
```